app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            eaten_points = self.appetite
+            return eaten_points
+        return self.appetite
```

Comment:
The `feed` method should return 0 if the animal is not hungry, rather than returning `self.appetite`. This ensures that the method behaves correctly by indicating no food was consumed when the animal is not hungry.